bazel/BUILD.bazel

```diff
@@ -8,7 +8,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
why was it possible to remove all the python toolchain configurations?

Reply:
We don't seem to require the toolchain configurations. The `py_binary` targets will generated an entrypoint that finds a Python interpreter on the system it is executed on. The toolchain required during the build is probably similarly autodetected.